README.md

```diff
@@ -1 +1,33 @@
-# node-global-lab
\ No newline at end of file
+# Use REPL and Global
+
+## Objectives
+
+1. Use REPL
+1. Access global
+1. Understand scoping and explicit vs. implicit declaration
+
+## Introduction
+
+Understanding global and scoping will give you a good foundation for future development.
+
+## Instructions
+
+1. Enter Node REPL
+1. Print global object
+1. Narrow down to specific info of process (`global.process`)
+1. Create a global variable `user` explicitly with `global.user = {admin: false}`
+2. Create a global variable implicitly with `var account = {balance: 1000}` 
+3. Open a new REPL and check for values of user and account (ReferenceError)
```

Comment:
I would explain `ReferenceError` more in depth. prompt them to think about why they would get the error and what that means